### PR TITLE
[Interop2022] Set up interop-2022 dashboard

### PR DESCRIPTION
**Pull request by KyleJu**
Set up a interop-2022 dashboard and a new /interop2022 endpoint. The dashboard is mostly taken and tweaked from compat-2021

### CODE REVIEW

**Comment by KyleJu** [comment]
https://interop-2022-dot-wptdashboard-staging.uk.r.appspot.com]

**Comment by chrishtr** [comment]
@argyleink

File: webapp/routes.go [hunk 1]
```diff
@@ -44,6 +44,9 @@ func RegisterRoutes() {
 	// Dashboard for the compat-2021 effort.
 	shared.AddRoute("/compat2021", "compat-2021", compat2021Handler)
 
+	// Dashboard for the interop-2022 effort.
+	shared.AddRoute("/interop2022", "interop-2022", interop2022Handler)
```
Comment (foolip) [inline webapp/routes.go:48]: Can you add a dash here and anywhere else it's missing? There's no particular reason to have it or not, but the repo for the effort has it.

**Review by foolip — APPROVED** [review]

File: webapp/components/interop-2022.js [hunk 2]
```diff
@@ -0,0 +1,838 @@
...
+
+        #featureSelect {
+          padding: 0.5rem;
+        }
+
+        #testListText {
+          padding-top: 1em;
+        }
+      </style>
+      <h1>Compat 2021 Dashboard</h1>
```
Comment (argyleink) [inline webapp/components/interop-2022.js:215]:
```suggestion
      <h1>Compat 2022 Dashboard</h1>
```

**Comment by argyleink** [comment]
https://interop-2022-dot-wptdashboard-staging.uk.r.appspot.com/interop2022?feature=summary

File: webapp/components/interop-2022.js (hunk 2 above)
Comment (foolip) [reply webapp/components/interop-2022.js:215]: Interop 2022 even :)

File: webapp/interop_2022_handler.go [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	"net/http"
+
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+type interop2022Data struct {
+	Embedded bool
+}
+
+// interop2022Handler handles GET requests to /interop2022
```
Comment (foolip) [inline webapp/interop_2022_handler.go:17]:
```suggestion
// interop2022Handler handles GET requests to /interop-2022
```

File: webapp/routes.go (hunk 1 above)
Comment (foolip) [reply webapp/routes.go:48]: Got on the computer to suggest instead:

```suggestion
	shared.AddRoute("/interop-2022", "interop-2022", interop2022Handler)
```

**Pull request merged**